renderer/components/Settings/SettingsPage.js

```diff
@@ -11,27 +11,44 @@ import ZapLogo from 'components/Icon/ZapLogo'
 import SettingsForm from 'containers/Settings/SettingsForm'
 import SettingsFieldsWallet from './SettingsFieldsWallet'
 import SettingsFieldsGeneral from './SettingsFieldsGeneral'
+import SettingsFieldsSecurity from './SettingsFieldsSecurity'
 import messages from './messages'
 
-const SettingsMenu = ({ group, setGroup, ...rest }) => {
-  const items = [
+const SettingsMenu = ({ group, setGroup, isLoggedIn, ...rest }) => {
+  // Items accessible to unauthenticated users.
+  const anonItems = [
     {
       id: 'general',
       title: <FormattedMessage {...messages.fieldgroup_general} />,
       onClick: () => setGroup('general'),
     },
+  ]
+
+  // Items only accessible to authenticated users.
+  const authItems = [
     {
       id: 'wallet',
       title: <FormattedMessage {...messages.fieldgroup_wallet} />,
       onClick: () => setGroup('wallet'),
     },
+    {
+      id: 'security',
+      title: <FormattedMessage {...messages.fieldgroup_security} />,
+      onClick: () => setGroup('security'),
+    },
   ]
 
+  let items = [...anonItems]
+  if (isLoggedIn) {
+    items = [...items, ...authItems]
```

Comment:
consider just doing `items = items.concat(authItems)` or `items.push(...authItems)` so it's cleared that we just adding authItems here